computer_vision/cifar10_pytorch/distributed.yaml

```diff
@@ -1,19 +0,0 @@
-name: cifar10_pytorch_distributed
```

Comment:
Ah sorry, intended to rename but didn't commit the last change. Updated now. Thanks for the catch

Reply:
I see. I'd suggest keeping `distributed.yaml` as is and not renaming it. almost all other examples use this name, because we assume that training is the default mode. it's really abnormal to have a `distributed_somethingelse.yaml`, like this example does.

Reply:
Sure, that makes sense. I've undone the renaming of that file/experiment.